src/backend/.env.example

```
@@ -72,6 +72,19 @@ PREFER_REMOTE_DOCKER_IMAGE=true
 # Example: your-org/custom-nlrf:latest
 REMOTE_DOCKER_IMAGE=monkscode/nlrf:latest
 
+# --- Docker Compose Configuration ---
+# Image tags for docker-compose deployment
+# Set to specific versions (e.g., "monkscode/nlrf-fastapi:v1.0.0") or use "latest"
+FASTAPI_IMAGE_TAG=monkscode/nlrf-fastapi:latest
+BROWSER_SERVICE_IMAGE_TAG=monkscode/nlrf-browser-service:latest
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BROWSER_SERVICE_IMAGE_TAG key should go before the FASTAPI_IMAGE_TAG key

(UnorderedKey)